x/superfluid/keeper/msg_server_test.go

```diff
@@ -130,6 +130,79 @@ func (suite *KeeperTestSuite) TestMsgSuperfluidUndelegate() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestMsgCreateFullRangePositionAndSuperfluidDelegate() {
+	defaultSender := sdk.MustAccAddressFromBech32("osmo104algnpncvwvsrm24ds2v2uk34gepg07qj3jkn")
```

Comment:
Any reason we use this specifically and not just the suite.TestAccs ?

Reply:
I remember failing at some point using testAccs due to bech32 decoding, but coming back to it, I was wrong, fixed this to use TestAccs